assets/scss/core/_typography.scss

```diff
@@ -85,41 +85,52 @@ h5,
 h6 {
   font-weight: $dp-font-weight;
   color: $dp-color-darkgrey;
+  margin: $dp-spaces--lv0;
 }
 
 h1 {
-  margin: 0;
-  font-size: $dp-sizing--lvl7;
-  font-weight: $dp-font-weight-bold;
-  line-height: $dp-spaces--lv6;
   font-family: $dp-font-family;
+  font-size: 30px;
+  line-height: $dp-spaces--lv6;
+  padding-bottom: $dp-spaces--lv4;
+  font-weight: $dp-font-weight-bold;
 }
 
 h2 {
-  margin: 0;
   font-size: $dp-sizing--lvl6;
-  line-height: 28px;
-  font-family: $dp-font-family;
+  line-height: $dp-spaces--lv5;
```

Comment:
this changes in the titles tags affect all the webapp?

Reply:
It probably affects in a general way, that's why I have to check the webapp well before merging